docker-hub/api/latest.yaml

```diff
@@ -18,37 +19,93 @@ info:
     Docker provides an API that allows you to interact with Docker Hub.
     Browse through the Docker Hub API documentation to explore the supported endpoints.
+
 tags:
-  # Primary objects
-  - name: Authentication
+  - name: rate-limiting
+    x-displayName: Rate Limiting
+    description: |
+      The Docker Hub API is limited on the amount of requests you can perform
+      per minute against it.
+
+      If you haven't hit the limit, each request to the API will return the
+      following headers in the response.
+      - `X-RateLimit-Limit` - The limit of requests per minute.
+      - `X-RateLimit-Remaining` - The remaining amount of calls within the limit
+      period.
+      - `X-RateLimit-Reset` - The unix timestamp of when the remaining resets.
+
+      If you have hit the limit, you will receive a response status of `429` and
+      the `X-Retry-After` header in the response.
+
+      The `X-Retry-After` header is a unix timestamp of when you can call the
+      API again.
+
+      *Note: These rate limits are separate from anti-abuse and Docker Hub
+      download, or pull rate limiting. To learn more about Docker Hub pull rate
+      limiting, see [Docker Hub download rate limit](https://docs.docker.com/docker-hub/download-rate-limit/).*
+  - name: authentication
     x-displayName: Authentication
     description: |
-      Most Docker Hub API endpoints require you to authenticate using your Docker credentials.
-      
+      Most Docker Hub API endpoints require you to authenticate using your Docker credentials before using them.
+
       Additionally, similar to the Docker Hub UI features, API endpoint responses may vary depending on your plan (Free, Pro, or Team) and your account's permissions.
       To learn more about the features available in each plan and to upgrade your existing plan, see [Docker Pricing](https://www.docker.com/pricing).
-  - name: Images
-    x-displayName: Advanced image management
+  - name: access-tokens
+    x-displayName: Personal Access Tokens
+    description: |
+      The Personal Access Token endpoints lets you manage personal access
+      tokens. For more information, see
+      [Access Tokens](https://docs.docker.com/docker-hub/access-tokens/).
+
+      You can use a personal access token in place of a password in the
+      [Docker CLI](https://docs.docker.com/engine/reference/commandline/cli/) or
+      in the [Create an authentication token](#operation/PostUsersLogin) route
+      to obtain a bearer token.
+
+      ### Scopes
+
+      For each scope grouping (in this case "repo"), you only need to define 1
+      scope as any lower scopes are assumed. For example: If you define
+      `repo:write`, the API assumes the scope of both `repo:read` *and*
+      `repo:public_read` as well. If you were to define both `repo:write` *and*
+      `repo:read`, then `repo:read` is assumed by `repo:write` and ignored.
+
+      ***Reminder: Personal access tokens should be kept secret like a password
```

Comment:
Suggest: Treat your personal access token like your password and keep it secret. You cannot retrieve your token after  it is generated.

Can we also use a Note here instead of a reminder? Here's the format we use in Docs, but not sure if Swagger can handle this format

```
> **Note**
>
> <text>
```

Reply:
I can update the text. Notes don't work in swagger